src/components/Envelope.vue

```diff
@@ -455,7 +456,6 @@ list-item-style.draft .app-content-list-item-line-two {
 }
 
 .icon-reply {
-	background-image: url('../../img/reply.svg');
 	-ms-filter: 'progid:DXImageTransform.Microsoft.Alpha(Opacity=50)';
 	opacity: 0.5;
```

Comment:
should i change the opacity .25 same as attachment?

Reply:
Whatever makes the look the same :wink: